Makes cult more robust

ChangelingRain opened the pull request:
:cl: Joan
rscdel: The Teleport Other, Veil, Disguise, and Time Stop runes have been removed.
rscdel: You can no longer have more than one rune per tile.
rscadd: Cultists can now communicate with all other cultists via the Communion verb, instead of requiring a tome.
rscadd: Replaced the Bind Talisman rune with the Summon Talisman rune, which allows the cultist to summon a talisman of choice instead of having to place a rune nearby and have paper on the rune.
wip: The Summon Talisman rune takes sightly longer to create and does slightly more damage on creation than other runes.
rscadd: Runes now tell cultists what they do on creation.
rscadd: Reading the tome will tell you basically everything about anything, including constructs, soulstones, runes, and armor.
experiment: You can examine runes to check what they do, as well as any other special effects or notes.
experiment: You can also examine soulstones and construct shells to see what you can do with them(as a cultist or wizard)
rscadd: Releasing a shade from a soulstone now allows you to reuse that soulstone for capturing shades or souls, instead of preventing the stone's use forever if the shade dies.
rscadd: The Teleport rune now allows cultists to select which rune to teleport to, and teleports everything on it.
tweak: Creating the Summon Nar-Sie rune does less damage and takes less time.
tweak: The Fabricate Shell rune now requires 30 metal sheets instead of 5 plasteel sheets.
tweak: The Reveal rune is now the Obscure Runes rune, which will hide or reveal nearby runes and spirits, depending on what state it is in. Examine it to check.
tweak: The Electromagnetic Disruption rune is slightly stronger.
tweak: The Blood Boil rune no longer sends every viewer directly into critical; instead, it does enough damage that one or two hits will knock them into critical and briefly stuns them. The rune itself now creates a small patch of fire when invoked.
tweak: The Drain Life rune now drains from each target on it, but has a slight cooldown.
tweak: The Manifest Spirit rune now notifies ghosts on creation.
tweak: Talismans of Veiling/Revealing will hide runes on the first use and reveal on the second.
tweak: You can now attack fellow cultists with a Talisman of Arming to arm them with robes and a sword.
tweak: The Talisman of Electromagnetic Pulse is slightly weaker.
/:cl:

ChangelingRain committed:
fancy cult changes

LatD commented:
Is this alternative to #16778 ? How do these differ?

KorPhaeron commented:
>3 different people trying to do cult balance at once without coordination

Why

ChangelingRain commented:
It is effectively an alternative; it differs mainly in the method of talisman production; here it's smack rune with paper, in that pr it's make specific rune, get talisman of choice. There's not much actual difference in speed/result.

Most of the important stuff is smoothing out new player experience; the comm verb is how all cultists communicate, making a rune says what it does, and teleport isn't random/doesn't rely on arbitrary keywords for proper use.

Cheridan commented:
>>3 different people trying to do cult balance at once without coordination

>Why

IT KEEPS HAPPENING
I TOLD YOU ABOUT CULT BRO

ChangelingRain committed:
nodupes

PKPenguin321 commented:
I like this one the most, it makes way more sense than robustin's and encompasses the narsie rune change from the other PR. Best of the three :+1:

ChangelingRain committed:
fix

swankcookie commented:
Definitely best of the three
:+1:

duncathan commented:
These changes aren't all mutually exclusive.

Robustin commented:
I agree, which is why I'm pretty damn sick of the same person behind "Lets close your oldcult PR so I can lecture you in IRC about the merit of replacing stun talismans with conversion talismans" dropping a competing PR along with Shadowlight within 12 hours of me putting out my update.

There are merits to some of these changes but they are minor and easy to add to what I've done. The bigger differences are where, as expected, I massively disagree:

1) My PR cuts more of the crappy runes and actually brings the rune list down to a sensible ~10 runes instead of leaving people with closer to 20 runes that perform duplicative functions.** In fact this PR ENTRENCHES the complete rune bloat by preventing me from moving some of the lesser used runes into talismans where they belong.** Now we're stuck with a largely useless stun rune because you still need them for talismans under this system. 

2) My system is designed to make it friendly to new players. There's still a lot to do in the regard but this PR defeats most of that effort. Having players smack runes with paper is about as unintuitive as the old system but with the added bonus that now your tome doesn't even HINT that this system is possible. In my system the moment you whip out a tome and look at runes, you have a CLEAR PATH to making talismans that will guide you along every step. **Pooping out runes and then leaving players, who have to be told my other players that this mechanic even exists, to smack timestop, blood boil, summon cult, manifest, etc. only to realize the system has been turned into an arbitrary retarded rollercoaster of imbue'ing that does nothing to slim down all the cult menu clutter is just stupid. ** No, this goes into "lets place stun talismans with conversion talismans" with your cult ideas that should be put to sleep and never discussed again. 

3) YES 3 PR's ABOUT THE NAR SIE RUNE. HNNNGH. My solution still works the best with its shields but largely I'm just sick of this nitpicking, a month where I could have gotten feedback, nothing. Cmon. 

4) I CANT WAIT TO DRAW SOME RUNES, OBSCURE THEM, THEN DRAW MORE AND BE STUCK IN THE SHIT TOGGLE BETWEEN HAVING HALF MY RUNES VISIBLE AND HALF HIDDEN. OH I NEED TO SACRIFICE TIME TO REVEAL, BUT OH SHIT THAT HIDES THE BARRIER RUNE I NEED TO ACTIVATE BEHIND ME, OH WAIT BUT THEN THAT HIDES THE TELEPORT RUNE I NEED TO ESCAPE, GUESS I NEED TO TOGGLE THIS GODDAMN RUNE LIKE 8 TIMES JUST TO KEEP MY CULT BASE USABLE HNNNNNNNNNNNGH. Again, absolutely no reason not to just keep hide/reveal in talisman form and let cultists have that control over their runes without burdening them with another terrible rune that will only undermine anyone who tries to use them. 

The other changes are either super minor, address other runes that I have elected to remove, or they are changes I agree with and will probably implement once I stop having to deal with fighting all these goddamn challenger PR's. God damn do you know how frustrating it is for someone who has to spend HOURS to code this stuff just to have someone who could have stepped in and tried to help jump in afterwards and go say "Nah fuck ur stuff, I played cult once I think I know better than you".

swankcookie commented:
holy shit

ChangelingRain committed:
this IS grudge coding I'm just doing it effectively

ChangelingRain commented:
Okay fixed it; the reveal/hide rune now does so based on its state rather than the state of the runes near it, so it's always either revealing or hiding.

If a rune can be imbued, it'll give you a message on creation.

ChangelingRain committed:
Stun and Debilitate runes will trigger when crossed, deleting themselves.

ChangelingRain committed:
desc

Robustin commented:
At least we agree on that, your response on Timestop is almost exactly what I said to Shadowlight. It's either overshadowed by Bloodboil or it will get buffed until its even more broken.

For both though, getting 3 cultists to stand near a rune that isn't Nar-Sie is almost impossible, it's like herding cats, that's always been the trade-off for blood boil being really, really good... if you can actually utilize ~teamwork~ as a cult you'll be scary.

ChangelingRain committed:
stop timestop

Shadowlight213 commented:
I fully support this PR Best option.

ChangelingRain committed:
strange anti-mine bias

ChangelingRain committed:
i am blind

ChangelingRain commented:
Stun and Debilitate runes will now affect only the mob that crossed them, for the same duration as if the rune was used manually. Crossing them will also REMOVE the rune, so invisible stun mines are one per tile.

Shadowlight213 commented:
>At least compared to Shadowlight you actually put effort into this and that allows

Someone's going back to trialmin.

ChangelingRain committed:
ayy

ChangelingRain committed:
testing code

PKPenguin321 commented:
jesus dicks oldman talk about tl;dr

this PR is better than yours and that is that, especially since it actually lists what it changes instead of screaming them at us in 40 paragraph rants

AdamElTablawy commented:
How about making stun runes not useless by making them not stun cultists near the rune when activated and give it a bigger range?

Also, if one major issue is confusion about what happened to the imbuing mechanics, just under draw runes expand the menu from 'draw rune' and 'cancel' to 'draw rune' 'cancel' and 'what is it?' What is it providing a detailed description of what the rune does and if it can be imbued and what effects it has if imbued.

Also add a text to cultist start that tells you slapping a rune with paper can let you imbue the paper with the rune's power.

I disagree with giving cultists an uncounterable undetectable hivemind-it makes it impossible for sec to detain cultists if they're constantly screeching about being jn the brig.

I also disagree with facilitating construct spam-15 sheets is too little. 20 sheets or 25 sheets is fairer.

Also, last suggestion here-is it possible to give the 'manifest ghost' rune a message to ghosts when it is created like golem runes do, and have it light up if a ghost is on top of it?

Xhuis commented:
I've been on /tg/ for two years now, and the _single_ game mode that causes the most drama is cult. Even shadowlings pales in comparison.

That being said, I support this PR more than the other two. I don't know why the Teleport Other dune is being removed, though.

ChangelingRain commented:
Stun runes already don't stun cultists.

Adding another button to the rune creation menu isn't possible without rewriting it and I'm no good at html interfaces and have even less idea how to do tgui; it's a good idea, but I can't really do it.

I'll probably bump shell creation up to like 30 sheets.
On comms, I think I'll make it say instead of whisper, and maybe make it unusable if stunned/cuffed, so it's more obvious/easier to counter. Right now you can counter it by knocking the cultist out, but ways to do that for security include chloral lethal syringes and crit, and those aren't particularly ideal if you're planning to deconvert.
Making a manifest ghost rune alert ghosts is super easy and I'll do that.

ChangelingRain committed:
manifest now notifies, fabricate takes 30 sheets, emp stronger

swankcookie commented:
>You can meme up blob and other crap, but god damn at least consult me about cult shit instead of playing these stupid games.


>consult me

The entitlement is real

Robustin commented:
The only entitlement is on the part of the coders who think they should be able to run wild over every aspect of this game without getting any feedback from the people who actually play. 

Having my oldcult PR closed, being dragged into IRC and told "this is how you're going to do things even though you've played more cult in a month than we have in our lifetimes" is the epitome of everything thats wrong with the way balance is done here. 

The sheer amount of politicking that has to be done just to get your way around here is just awful and for the place that should be where coders get the most comprehensive and intelligent discussion, all I see is THUMBS UP TO LIKE and one-sentence shitposts from OTHER people who have no clue how these modes play out in reality.

The fact that a coder is trying to bring back LE INVISIBLE STUNMINE MEME and people are going "YEA THIS IS GOOD I LIKE THIS" is just a show of what a farce these conversations are. Its not just the change itself, but the fact that Joan and the peanut gallery are all even CONTEMPLATING that this could be an improvement to cult just makes me gag. 

Making PR conversations only about the code aspect and move the balance talk to Feedback and you'll start seeing a huge change in how the "community" feels about these changes.

Shadowlight213 commented:
I think that robustin's idea with the shields protecting the rune drawer is a good idea actually.
We ran a testmerge of the changes last night and we literally had a perfect cult round. Joan said that most of their changes didn't come into play, with the two that affected the round most being innate comms and the nerf to drain life.
The biggest issue I saw the cult facing was actually trying to draw the summon rune and not getting bumped around. The shields would help with that.

Also, mekhi joining as captain when the cultists were in the middle of drawing the narsie rune was hilarious.

swankcookie commented:
By "play[ing]  more cult in a month,  than....",  do you mean on a private testing server?  By yourself? I don't know,  it just seems like,  to me, that's not 100 percent representative. 

>The sheer amount of politicking that has to be done just to get your way around here is just awful

Your acting like you have to give someone head just to get a PR through,  when the truth is you just need yours to be the best option,  which,  when your dealing with 2 other PRs in regards to a complicated game mode,  isn't so simple.

ChangelingRain commented:
In reference to that round, the ONLY changes that came into play were the innate comms, the blood boil making fire thing(one use of blood boil, a few cultists being on fire for a very short time), the drain life cooldown(someone trying to instakill with it and it not doing that), and the nar-sie rune change(barely, it just sped it up a little).

AdamElTablawy commented:
What about sending out an alert if a rune is drawn, somewhere? 

An important rune, that is.

So say if you draw the summon nar'sie rune it broadcasts to all cultists 'nar'sie rune has been drawn at X', but it doesn't spam with endless alerts about where convert runes were drawn.

swankcookie commented:
That's a good idea

PKPenguin321 commented:
this is a good PR, oldman is just salty that he's been outdone. ignore him

KorPhaeron commented:
Great now I have to actually read both of these

Robustin commented:
@AdamElTablawy 

Thats been a part of oldcult since it came back....

Moving on. This is still oldcult we're dealing with. The fundamentals between our PR's hasn't change. I'd say 80% of cult rounds would play out the same under either of our PR's. I don't think sporadic test merges are going to prove much. 

Let's compare the remaining differences between our PRs:

1) My PR trims more runes in an effort to make cult more newbie-friendly. Joan trims some runes but expands the power of others. Neither of us have strong feelings about timestop but I am adamant that the return of stunmines is unacceptable. 

2) My PR summons all talismans through a single rune. This means that even a baby cultist won't have to ask "how do I get talismans" anymore. Considering that cult balance is completely based on the power of talismans, I felt that this was a great solution. It helps remove one of the cult's biggest weaknesses (deadweight newbie cultists who are more likely to get caught than actually do something useful).

Joan's PR makes talisman construction a matter of hitting your rune of choice with paper. I felt this was the inferior option because a baby cultist is not going to know that mechanic and there won't be any obvious indicators telling them that. Not even roundstart text is as helpful as having "CREATE TALISMAN" staring a cultist in the face every time they prepare to draw a rune. Joan's method is probably better than the current system, but I'm very certain that mine is the superior method between our PR's. As I already pointed out, only roughly half of runes are even capable of being made into talismans and right now under Joan's PR the only way for a newbie cultist to find that out is by drawing up the runes and inspecting or hitting them to see which ones stick. With my PR you've just got this nice simple menu that pops up with a straightforward list of what talismans are available to the cult, no guesswork involved. 

3) Joan's PR makes the nar sie rune appear faster and cost less health. Joan's Nar Sie rune appears in about 5 seconds. My PR also makes it short, to 40 seconds, but also summons a 3x3 grid of 60hp ~~~thematic~~~ barriers to prevent anyone from bumping the summoner. This gives the crew a chance to act against a cult rushing out nar-sie but also means that the summoning won't be interrupted by accident. This is another aspect where I am extremely confident that my approach will produce better rounds. 

4) Joan makes cult chat akin to a hivemind that requires no items or rune interaction to communicate. I keep the system that requires a tome to communicate but allows for 'emergency' communications without a rune or book if you're unrestrained, at the cost of a moderate amount of brute damage that is likely to draw blood, and attention. My issue with Joan's version is that you might as well cut the entire deconversion system too. When cult has a universal hivemind, standard procedure is for security to execute cultists on the spot because they will be shouting for help the moment sec they are arrested and with cult able to summon their pals from anyone in the world... its too much of a risk. I believe that my approach gives cult more depth and I'm generally just not a fan of cultists being able to waltz around and chat with their entire team effortlessly. The cult should be SECRETIVE and for me that means at least pulling out a tome if you're going to talk in the cult chatroom. I addressed the issue of emergency communication in my PR by adding a new prompt that will make sure new cultists dont cut themselves up at roundstart trying to figure out how to talk. 

5) As an extension of #1: my PR moves certain niche runes into talisman status. There's no reason to have summon tome runes AND summon tome talismans. To ease tome distribution, my PR gives a tome with every successful conversion, if you still need more you can draw a talisman that can summon up to two. This makes the cult system MUCH, MUCH cleaner. Instead of having to scroll my mouse 3 times so I can try to find the EMP rune as its lodged between 'drain life' and 'deafen'. The entirety of cult magic is now neatly condensed into a manageable set of runes and a more robust talisman creation/selection system.

6) I also add color-coding to talismans. Since there a couple new additions, it makes more sense than ever to give cultists an easy way to distinguish between different talisman types/ordinary paper/etc. There is also a new "Horror" talisman that will cause the victim to hallucinate for a few minutes. Gives cultists a way to subtly undermine security without having to stun and carve their face off. 

When it's all said and done I'm very proud of the work I've done in my PR and I hope that with careful consideration people will understand the logic behind the choices I've made.

ChangelingRain commented:
Stunmines were only awful in newcult because they were unremovable and permanent, and thus were more or less wall runes but better, because they were passable by cultists and nobody else.

Runes that can be made talismans throw a message when created. I'll make it bigger; it's already explicit "HIT THIS WITH PAPER" message.

The nar-sie rune takes 35 seconds with my changes.

I'll make cult chat unusable if restrained(though not stunned) so you can just keep the cultists cuffed to keep them from talking.

I'm just gonna reorder all the runes in the menu so they're slightly more ordered.

Color-coded talismans basically ruins the stealth aspect of it, but it's good for cultists because it lets them check which each is. I'm kind of on the fence here; I have talisman sprites if we want to totally drop the stealth aspect and would like to use them, but I'm not sure the loss of stealth is totally worth it.

KorPhaeron commented:
>Color-coded talismans

Joan you have an addiction

ChangelingRain commented:
Robustin did those I made actual sprites!

ChangelingRain committed:
this makes the diff absolute dongs I'm sorry

Robustin commented:
I'll test your talisman approach but my belief is that as a result of us giving red flavor text to every kind of antag interaction, people are going to filter that stuff out. 

I just don't understand the opposition to my approach here. The numbers for time to imbue can all be tweaked, but having a nice simple menu of "here's all the talismans available to you, pick one" seems much more preferable too "carve up runes and look at the text for each one to tell if you can imbue it onto a talisman."

ChangelingRain commented:
Okay the red flavor text thing is a legit concern, actually. Part of it is people not reading shit, but that's a good point.

Robustin commented:
Ok tested it:

1) You've got a serious bug with rune-making. I can open the menu to scribe a rune, walk away, select a rune, and the rune will draw in the tile where I originally opened the menu. 

2) Talisman making was just as unattractive as I thought it would be. I mean look at these two images, is there even a serious debate here?!

3) I'm glad you took the feedback on the obscure rune, I may end up using it too in talisman form (I will credit you on the changelog for the stuff I end up adding from you). 

4) Invisible stunmines WHY WHY WHY. 

5) Our communication is almost identical now, glad to see you can't use it while restrained. I still prefer the tome + emergency system for the thematic stuff but it's become a really small distinction at this point. I don't feel strongly about it one way or another.

ChangelingRain committed:
where are you now, cultist?

ChangelingRain commented:
At... what images?

PKPenguin321 commented:
>2) Talisman making was just as unattractive as I thought it would be. I mean look at these two images, is there even a serious debate here?! http://imgur.com/a/u6DYG

i do not see the issue?

Robustin commented:
@PKPenguin321 

If you look at those pictures and say "Gee I don't see the difference between having to draw up 20 something runes and inspect them all manually to tell which ones I can talisman" versus "Holy shit here's a menu with all your damn choices in one simple list!" I don't know what's going on in that head of yours. 

How about I make a restaurant where guests are given a menu with what's available at that time of day and they pick a choice... OR I write down the name of every entree we sell at Breakfast/Lunch/Dinner and throw them all of the floor face down, and let you scramble around for 5 minutes trying to keep track of what you can buy for dinner.

PKPenguin321 commented:
i mean realistically you already know what runes you can invoke into a talisman

Robustin commented:
Realistically I'm not overhauling the talisman system for ME, I made it for the 95% of other players who literally never make a talisman the entire round because its a really confusing system for inexperienced players.

PKPenguin321 commented:
put a little "(can be made into a talisman)" next to the rune's name on the draw rune menu, then? it still won't look as pretty but at least it's still straightforward and keeps the on-ground rune functionality

AdamElTablawy commented:
okay robustin you're on about invisible stunmines

where are the invisible stun runes here

Robustin commented:
I dunno, go read the code for it, I deleted it in my PR.

I just confirmed it through testing her PR. Stun runes work as mines and they work when obscured.

ChangelingRain commented:
You can hide stun runes with the obscure rune and they'll stun people on crossing, but they only last for one stun; it's generally better to invoke them or use a stunpaper, given that you can't stack multiple on a tile.

optimumtact commented:
If you bring back stunmines I'm going to remove cult

Xhuis commented:
Stun mines already exist. They're called "banana peels" and any competent botanist can produce hundreds.

Robustin commented:
Peels aren't invisible, peels can be inspected by regular crew, peels can be picked up by anyone, peels slip cultists and non-cultists all the same, peels represent their danger even at a glance, and peels require some concerted effort to produce.

Stunmines can be invisible, regular crew have no way of telling that a rune is a stunmine, they can't be removed by anyone except the station's 1 null-rod user, stunmines don't affect cultists, their danger is not apparent at a glance, and stunmines can be produced at roundstart with about 3 seconds per rune. 

It's a stupid comparison and I'm tired of having to point this kind of stuff out.

AdamElTablawy commented:
You can fill a fucking room with invisible stunmines and cheese summon narsie if the chaplain is dead, because you can cover all of maint next to the cult base and the cult base in invisible stunmines.

If you MUST bring back stun mines, make them also trigger if a cultist steps on them and make them unable to be invisible-runed. Otherwise they're cancerous and overpowered, and no amount of comparison to banana peels equates to a spammable roundstart invisible trap that only affects nonantags with no risk of slipping the cult like a banana peel can.

ChangelingRain commented:
You can surround the nar-sie rune with wall runes and summon the entire cult in to cheese summon nar-sie if the chaplain is dead.
The stun mines disappear after one stun and are less effective than a stunpaper or just actually invoking the rune.

ChangelingRain committed:
stun and debilitate are just flat out fucking weaker when crossed stun's barely even worth it now just fucking stunpaper them

Robustin commented:
Yea but if wall runes are up:

1) The wall runes don't kill me, hurray
2) I will know they exist by their obvious effect and try to come up with a countermeasure like welderbombs, chem nades, flamethrowers, or whatever else will disrupt the ritual whereas by the time I realize a stun rune is there, I'm getting talisman'd/baton'd/shot to death. 
3) Its not just about nar-sie, even if the cult is nowhere near ready to summon, it can still just cake maint with runes and destroy anyone who tries to stop them.

ChangelingRain commented:
ACTUALLY INVOKING THE RUNE STUNS FOR LONGER THAN CROSSING IT AND IT STUNS EVERYONE NEARBY HOW IS TAKING THE TIME TO PLATE THE AREA IN WEAKER MINES SIGNIFICANTLY BETTER THAN JUST INVOKING THE RUNE IN THIS HYPOTHETICAL SIEGE ON CULT BASE SCENARIO

Robustin commented:
The fact you don't understand why after all this time is something I can't help and all the more reason to believe that this isn't the right direction for cult. 

There's such a massive difference between something you have to directly invoke when someone is in (X) range versus an autonomous defense of invisible fuckmines that you can just sit in a locker nearby and assblast anyone who had the audacity to walk in maint. 

I have a better solution for this anyway that I'll test out when I get a chance.

AdamElTablawy commented:
Look the issue is that even if nobody's watching people are gonna be stumbling and tripping all over themselves walking on eggshells they can't even see and god help them if someone broke a window to space because lol drifting.

Just remove the invisibility function of stun mines then so you can't make them invisible. Also if a cultist IS nearby to take advantage of them then they're overpowered as shit.

ChangelingRain committed:
this is free popularity

ChangelingRain commented:
Now stun mines are gone, until they return later, in the future(:tm:)

ChangelingRain committed:
testing is for squares
i am not square

Robustin commented:
So now we're down to: 

1) Making imbue a simple menu v. paperslapping runes

2) Streamlining runes v. Leaving them as-is (you cut a few but still have like 8 more than me)

3) A bunch of other minor issues comparisons Tome communication v. Hivemind communication that functions almost identically to Tome communication, Nar Sie Shielding, EMP changes, color coding talismans, etc...

The first 2 are really all that matters between our PR's now since that's where they're the least compatible. My stance has been steadfast. Across the hundreds of cult rounds I've played, there's been one singular constant: Newbie cultists don't contribute anything, get caught, and act as a net liability to the cult. That's why I made my PR in the first place, to hopefully make a big advancement in making cult something that's actually playable for someone who hasn't memorized every aspect of the game mode. This image continues to remain relevant and it shows the superiority of my approach:

http://imgur.com/a/u6DYG

duncathan commented:
I much prefer Robustin's imbue rune to slapping runes with paper. He's right; it's more immediately understandable and user friendly.

ChangelingRain committed:
ezrunes

ChangelingRain committed:
absentmemed

ChangelingRain committed:
really, really absentminder

ChangelingRain commented:
Alright, adjusted talisman creation to use a rune; it uses no paper and is instant, but has a larger health cost(2 as compared to 0.1) and takes very slightly longer(6 sec as compared to 5) than other runes.

I'm not totally sure on this because it's basically a menu for 6 talisman types, because I removed disguise talismans and combined hide/reveal into one.

ChangelingRain committed:
Fixes #16856

Robustin commented:
Right so we're rapidly approaching the point where we ask? What's the difference?

I still cut more runes in mine and it addresses your issue with "too few talismans" since I added Talisman of Horror and would want to incorporate a multi-use obscure talisman instead of separate hide/reveal. But I'd still want to fight for scaling EMP rune, colored talismans, moving more runes into talismans as I did, etc... and part of me would want to push ahead with mine simply because it was first, it was my initiative to take cult in this direction, and it would give me an incentive to continue making improvements without having to worry about this happening over and over.

ChangelingRain commented:
Well, I'm not totally sure about differences right *now*, but basically this is the first of what might become a bunch more changes, most of which I don't want to put in this pr.
Some might be controversial, like mine runes, some are effectively feature creep, and there's a few things that are more prototype. (I'll totally expand on any of those things if you ask but what they are isn't directly relevant to the question besides the possibility of you planning things like them)

Robustin commented:
Which still brings us back to the issue of deciding which PR gets merged. 

Future PR's can be resolved on their own merits. We're at the point where the final lines are drawn and the current choices are clear. I say that the oldcult-haters got their chance at bat from Nov.-February. Now I deserve to take a crack at streamlining the experience. I worked hard to bring back oldcult in a more palatable form and I worked hard on my PR. You already have virtually uncontested control over several other sections of code that are crucial to balance right now and it's still worth nothing that my PR came first.

I don't see any compelling reason for taking this PR over mine, but we need a way to move forward on this one way or another.

ChangelingRain commented:
I'd like to note I don't have 'uncontested control' over shit. The assumption is that I do because I'm the only person doing stuff to those areas of code but that's just not how open source works.
I might have tacit authority on those things but that's not actually control.

kilkun commented:
make nar-sie a level 6 singulo without emp again (now that singulag is fixed there's no real reason for the pussy nerfed version) and reduce the required count for conversion runes to 1 and this pr is 10/10. Cult has never really been good, but neo-cult is especially bad and I don't feel Forceful's PR addresses the issues in a constructive way, at least in the tiny ways it differs from this one. Cutting even more of cult's abilities doesn't fix it. It's a defensive, stealth, conversion antag. The triumverate of terrible ideas, combined with the fact that cult is literally a half-baked gamemode made out of the scraps of another's assets, only it lost literally all of the great gimmick opportunities because it never evolved beyond being half of a gamemode.

I think the talisman changes are a good start, as giving them more on-the-go abilities improves their offensive capability in a way that a shitty esword and shitty armored suit doesn't, but neither PR will make cult fun. Lovecraftian cosmic horror is a good fit for ss13, but the gamemode doesn't reflect the variety and strengths of cosmic horror. It's just wizards who cut themselves while saying nonsense before a pussified cosmic deity shows up and you have to wait 15 minutes for the round to end. A round that you spent bored out of your mind because someone unironically thought a gamemode that boils down to stealth revs with absolutely no offensive capabilities outside of a shitty esword and shitty set of armor that anybody instantly recognizes as ez valids was a good idea.

Robustin commented:
The fact you don't know that Nar-Sie summons a 2-minute shuttle goes to show you really haven't played cult recently. 

Also 
>Stun + Mute talismans
>Juggernauts
>Wraiths
> No offensive capability

ChangelingRain commented:
>Bloodboil an entire shuttle
>>No offensive capability

It has ample, but not always well-used, offensive capability. Swords are basically a backup weapon when you can have a guy draw a rune that crits everyone on the shuttle.(yes with my changes it's not quite crit but still)

kilkun commented:
@Robustin I don't pay attention to the shuttle time, but it's still a long ass time to wait no matter what.

Also juggers and wraiths are no match for a competantly led sec team. And one use melee stuns aren't shit. It's better to make a stun prod and use that + an emp to get a lad than waste your talismans.

@ChangelingRain Bloodboil requires a chokepoint, it's not like you can instantly walk into the shuttle and place one down. More than likely you are going to get lasered on sight. It's a very valuable and robust rune (one that made cult unwinnable when it was removed for a very short time) because it is imperative to base defense, but using it offensively requires great care and setup ahead of time, because it's not like you can plop one down while you are being actively attacked.

ChangelingRain committed:
fixed multiple teleport runes

Cheridan commented:
>Pr with both the Ready for Review/Merge tag and the Do Not Merge tag.
:rage3:  :gun:

AdamElTablawy commented:
the issue I believe with cult's offensive ability is that it is one of the only antags that is ENTIRELY dependent on teamwork for ANY offensive capability

at least with revs, gangs, ect you can use your antag shit solo as well as in a team, but a cult will get fucked if trying to use any of their offensive shit without anyone nearby, which is probably why it never gets used-it's very hard to get 3 people to stand around a circle in the middle of combat to blood boil.

ChangelingRain committed:
Examining soulstones and construct shells gives actual info, good lord

lordpidey commented on code/game/gamemodes/wizard/soulstone.dm:
```diff
@@ -96,6 +102,15 @@
 
+/obj/structure/constructshell/examine(mob/user)
+	..()
+	if(iscultist(user) || iswizard(user))
```
how about letting ghosts see it too?

ChangelingRain replied:
Sure.

ChangelingRain committed:
shadespam is a valid tactic, guys

ChangelingRain committed:
explicit texts

ChangelingRain committed:
detail EVERYTHING in the tome

ChangelingRain committed:
Noone reads this but update it anyway, it wouldn't do to be outdated

ChangelingRain committed:
brief stun

phil235 commented on code/game/gamemodes/cult/ritual.dm:
```diff
@@ -239,24 +232,28 @@ This file contains the arcane tome files as well as innate cultist emergency com
-				return
-	if(!do_after(user, 50, target = get_turf(user)))
+			priority_announce("Figments from an eldritch god are being summoned by [user] into [locname] from an unknown dimension. Disrupt the ritual at all costs.","Central Command Higher Dimensionsal Affairs")
+	if(!do_after(user, initial(rune_to_scribe.creation_delay), target = Thenewturfyouwalkedto))
```
you don't need to specify a target, the turf isn't going anywhere...

ChangelingRain replied:
From checking the code it appears I need a target for a progress par to appear; I'll test and see if it works without one but I think it does need it.

ChangelingRain replied:
Yup, it does need a target for a bar. It's a choice of either user or turf so I'll go with this.

phil235 commented on code/game/gamemodes/cult/runes.dm:
```diff
@@ -414,238 +309,297 @@ var/list/teleport_other_runes = list()
-		user << "<span class='cultitalic'>The sacrificial target must be dead!</span>"
+
+	var/input_rune_key = input(user, "Choose a rune to teleport to.", "Rune to Teleport to") as null|anything in potential_runes //we know what key they picked
+	var/obj/effect/rune/teleport/actual_selected_rune = potential_runes[input_rune_key] //what rune does that key correspond to?
```
I'm not sure, but would this runtime if you cancel? The input_rune_key ends up being null and you call "potential_runes[input_rune_key]".

ChangelingRain replied:
Nope, no runtimes from doing that.

ChangelingRain committed:
this is an actual improvement

phil235 commented on code/game/gamemodes/cult/runes.dm:
```diff
@@ -414,238 +309,297 @@ var/list/teleport_other_runes = list()
-	empulse(src, 4, 8) //A bit less than an EMP grenade
+	playsound(T, 'sound/items/Welder2.ogg', 25, 1)
+	qdel(src) //delete before pulsing because it's a delay reee
+	empulse(T, 5, 10) //A bit less than an EMP grenade
```
Did you test that this work and the pulse if called even after the qdel(src) ?

ChangelingRain replied:
It works, yes. i could spawn it off, I guess, but it's mostly a style thing.

ChangelingRain commented:
here is oldman's fancy poll vote joan 2k16 https://tgstation13.science/phpBB/viewtopic.php?f=10&t=6476

Razharas commented:
Robustins one was first to get on my eyes so tough luck
